bgs/admin.go

```diff
@@ -647,3 +647,114 @@ func (bgs *BGS) handleAdminRequestCrawl(e echo.Context) error {
 
 	return bgs.slurper.SubscribeToPds(ctx, host, true, true) // Override Trusted Domain Check
 }
+
+// Whoooo are the users in your neighborhood?
+type ApiKey struct {
+	// ApiKey goes in header: `Authorization: Bearer {}`
+	ApiKey string `json:"key"`
+	// Priority; more is more (unkeyed == 0)
+	Priority int32 `json:"prio"`
+}
+
+// set by
+type ApiKeySet struct {
+	Keys []ApiKey `json:"keys"`
+}
+
+// ApiKeys is the in-memory version
+type ApiKeys struct {
+	Keys map[string]ApiKey `json:"keys"`
+}
+
+func NewApiKeys() *ApiKeys {
+	out := new(ApiKeys)
+	out.Keys = make(map[string]ApiKey)
```

Comment:
Can we do this more idiomatically?
```suggestion
	out := &ApiKeys{Keys: make(map[string]ApiKey)}
```